### DISCUSSION TITLE
Is it possible to prevent the screen from going into screensaver/sleep?

### DISCUSSION THREAD

**Discussion by abique** [rust-windowing/winit#3780] [Q&A] [answered]
Hi,

Is it possible to prevent the screen from going into screensaver/sleep?

Thanks :+1:

**Comment by madsmtm** [comment]
Not by using only Winit, no, and this will likely remain out of scope: https://github.com/rust-windowing/winit/issues/616#issuecomment-2222955313

**Comment by kchibisov** [comment, marked as answer]
There will be an extension in the future, see https://github.com/rust-windowing/winit/issues/3351 for the progress. 

It does require windowing system, so not out of scope, at least on Wayland.

**Reply by abique** [reply]
Thank you very much! :-)
